blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -105,9 +106,13 @@ export class RunDetailsPipeline extends Component {
     }
 
     componentWillUnmount() {
-        if (this.listener.sse) {
-            sseConnection.unsubscribe(this.listener.sse);
-            delete this.listener.sse;
+        if (this.listener.ssePipeline) {
+            sseConnection.unsubscribe(this.listener.ssePipeline);
+            delete this.listener.ssePipeline;
+        }
+        if (this.listener.sseJob) {
+            sseConnection.unsubscribe(this.listener.sseJob);
+            delete this.listener.sseJob;
```

Comment:
In fact, considering that the code is using the same listener instance in both cases (`this._onSseEvent`), then I'm wondering what's going to happen here.